frontend/packages/dev-console/integration-tests/features/pipelines/event_tab_pipeline_run_page_ODC_4314

```diff
@@ -0,0 +1,56 @@
+Feature: Events tab in Pipeline run and Task run details pages
+	As a user, I want to see Events tab in Pipeline run and Task run details pages
+
+Background:
+    Given Openshift Pipelines operator is installed
+
+@regression
+Scenario: Events tab in pipeline run details page of administrator perspective
+   Given user is in administrator perspective
+   And atleast one pipeline run is present
+   When user clicks on pipeline tab in navigation menu
```

Comment:
```suggestion
   When user opens Pipelines page
```

We don't need to mention the navigation menu everytime

Reply:
It won't create a difference